zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/NoteManager.java

```diff
@@ -378,6 +386,24 @@ private String getNoteName(String notePath) {
     return notePath.substring(pos + 1);
   }
 
+  private boolean validateNotePathDoesNotExist(String notePath) {
```

Comment:
Just a small style change.
Please rename your method name to improve readability.
https://rules.sonarsource.com/java/tag/convention/RSPEC-2047

btw. +1 for the new Exception

Reply:
I have renamed inline with the feedback. Thanks